ember-scoped-css/src/lib/generateRelativePathHash.ts

```diff
@@ -1,5 +1,5 @@
 import md5 from 'blueimp-md5';
 
-export default function generateRelativePathHash(relativePath) {
+export default function generateRelativePathHash(relativePath: string) {
```

Comment:
this was converted to TS because I wanted `scopedClass` to be TS, so I could provide an overload type for it.